tests/unit/license_example

```diff
@@ -0,0 +1,24 @@
+{
+  "key": "mit",
```

Comment:
If all of this is in a license response, why does the `License` class have so few attributes?